5_smooshed_morse_code/easy/README.md

```diff
@@ -16,4 +16,6 @@ Por exemplo:
 "programmer" => ".--..-.-----..-..-----..-."
 ```
 
-
+Nota: A execução do teu programa não deve aceitar argumentos a não ser o próprio programa e um ficheiro com dados de input. O input pode também ser lido através do stdin até EOF. 
+Por exemplo em C:
+`./program input.txt` ou `./program < input.txt` 
```

Comment:
I think we should read always from stdin.

Reply:
This was a discussed with @ZePedroResende, @herulume and @0urobor0s. We decided to specify the input this way to allow some freedom but with limitations.